### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by bernhardmgruber**
Includes more platform agnostic build/install instructions and a note that Vc is now in maintenance mode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,19 +118,36 @@ $ mkdir build
 $ cd build
 ```
 
-* Call cmake with the relevant options:
+* Configure with cmake and add relevant options:
 
 ```sh
-$ cmake -DCMAKE_INSTALL_PREFIX=/opt/Vc -DBUILD_TESTING=OFF <srcdir>
+$ cmake -DBUILD_TESTING=OFF ..
```
Comment (amadio) [inline README.md:127]: How about making `BUILD_TESTING` off by default and adding `-DBUILD_TESTING=ON` to the CI?

Comment (bernhardmgruber) [reply README.md:127]: We could do that, but that could affect a lot of downstream users. Let's do that in a separate PR.

Comment (bernhardmgruber) [reply README.md:127]: New PR here: #325

**Comment by amadio** [comment]
Is this ready to be merged?

**Comment by bernhardmgruber** [comment]
Yes, please!

**Pull request merged**